AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -88,6 +90,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * An ASLayoutSpec subclass that can wrap a ASLayoutable and calculates the layout of the child.
+ */
+@interface ASWrapperLayoutSpec : ASLayoutSpec
```

Comment:
Can we introduce `layoutableThatFits` so we don't have to have a wrapper layout spec?